types/types.override.pb.go

```diff
@@ -1,7 +1,30 @@
 package types
 
-// XXX_WellKnownType is a hack -- please don't make a pattern out of this!
-// This is used to trick certain toolsets to treat this field like a WKT StringValue.
-// See https://github.com/golang/protobuf/blob/70c277a8a150a8e069492e6600926300405c2884/jsonpb/jsonpb.go#L157
-// and https://github.com/golang/protobuf/blob/70c277a8a150a8e069492e6600926300405c2884/jsonpb/jsonpb.go#L191
-func (*UUIDValue) XXX_WellKnownType() string { return "StringValue" }
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/protobuf/jsonpb"
+)
+
+// MarshalJSONPB overloads UUIDValue's standard PB -> JSON conversion
+func (m *UUIDValue) MarshalJSONPB(*jsonpb.Marshaler) ([]byte, error) {
+	return []byte(fmt.Sprintf(`"%s"`, m.Value)), nil
```

Comment:
does this have to be `Sprintf`'d? If so, how come?

Reply:
The Unmarshal strips double quotes and the Marshal re-adds them so that it isn't stored with the quotes. Sprintf just came to mind as the proper tool.

Reply:
oops.. didn't notice the quotes in there. You're all good :)

Reply:
%q instead of %s probably does what you want.